changelog/unreleased/vm-id-label.md

```diff
@@ -0,0 +1,4 @@
+Change: Add VM ID to labels
```

Comment:
Maybe you change it a little bit? The last line always got to container links to reference like an issue or pull request.

```
Change: Add new label for virtual machine ID

We've added a new label to get the current ID of a virtual machine in the format provided by vCloud Director in the form of `urn:vcloud:vm:807b799e-c72f-4592-9acd-ccefefe92720`.

https://github.com/promhippie/prometheus-vcd-sd/issues/30
```

Reply:
applied in [8bb507d](https://github.com/promhippie/prometheus-vcd-sd/pull/31/commits/8bb507df716907a7279d67e92d7664cf918b5fd4)